cmd/fossa/cmd/test/test_test.go

```diff
@@ -0,0 +1,169 @@
+package test
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/fossas/fossa-cli/api/fossa"
+	"github.com/stretchr/testify/assert"
```

Comment:
[nit] Import groups should be: standard library, third party, first party.

Reply:
You're right, I relied on `go fmt`. I wonder why it ordered them wrong in this case.

Reply:
`go fmt` will do grouping, but isn't smart enough to understand group ordering.